src/frontend/components/avatar.js

```diff
@@ -2,8 +2,11 @@ import { LitElement, html, css } from "lit";
 import { when } from "lit/directives/when.js";
```

Comment:
in questa classe sono rimasti i colori di sfondo degli avatar di default, anche questi vanno gestiti tramite il servizio o le costanti